manifests/init.pp

```diff
@@ -126,6 +126,12 @@
     fail("${facts['os']['name']} not supported")
   }
 
+  if (($facts['osfamily' == 'RedHat') and versioncmp($facts['operatingsystemmajrelease'], '7') <= 0) {
```

Comment:
there is a `]` missing for `$facts[]`?

also both facts are legacy. Please use `os.family` and `os.release.major` instead.